istioctl/cmd/root.go

```diff
@@ -216,21 +221,24 @@ debug and diagnose their Istio mesh.
 	experimentalCmd.AddCommand(describe())
 	experimentalCmd.AddCommand(addToMeshCmd())
 	experimentalCmd.AddCommand(removeFromMeshCmd())
-
-	experimentalCmd.AddCommand(vmBootstrapCommand())
+	vmBootstrapCmd := vmBootstrapCommand()
+	deprecate(vmBootstrapCmd)
```

Comment:
@esnible Is this really going to be deprecated? I see it has been added recently https://github.com/istio/istio/pull/23126.

Reply:
During the August 25 UX meeting, it was felt this command is totally outdated and will be replaced with the new `workload-entry` subcommand.
I am not sure if anyone from Tetrate was there to confirm that, but we should deprecate it and see if anyone complains about the deprecation before the November release.